drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/AnalyzerProvider.java

```diff
@@ -42,6 +42,12 @@
 
     public AnalyzerProvider() {
 
+        factTypes.add( new FactTypes.FactType( "Applicant",
+                                               new HashSet<FactTypes.Field>() {{
```

Comment:
Unless you need to modify the `Set` later, `Collections.singleton(...)` will work just as well and will bring nicer code.